src/libraries/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/AddRefTests.cs

```diff
@@ -29,14 +29,6 @@ public void AddRef_ValidPointer_Success()
         }
 
         [Fact]
```

Comment:
It would be nice to have test coverage for the success cases on Unix as well, but I understand that it is not easy to add it at the moment. It should be easier to add once we have ComWrappers for Unix. I think it is ok to merge this with the missing test coverage and open an issue to add it later.

Reply:
#51072